endpoint/model.json

```diff
@@ -765,7 +785,20 @@
           "type": "array",
           "description": "The message groups that are supported by this endpoint",
           "item" : {
-            "type": "uri"
+            // New type: "relation", pointer to data inside of the xregistry
+            "type": "relation",
+            // Target "type", if omitted it can point to any type inside the registry
```

Comment:
oops, the spec has this as required not optional. Let's discuss along with https://github.com/xregistry/spec/issues/226

Reply:
agreed on 1/15